stone/lang/lexer.py

```diff
@@ -100,41 +102,31 @@ def test(self, data):
         'KEYWORD',
         'PATH',
         'DOT',
-    )
 
-    # Whitespace tokens
-    tokens += (
+        # Whitespace tokens
```

Comment:
I did not. It had trouble automatically figuring out that the initial `tokens` tuple was a Tuple[many strings], so I decided to jsut collapse it all together.

There's two solutions:
* add `# type: Tuple[str, ...]` (which i did elsewhere in this code review)
* just collapse them all together

happy to go back to the original approach :)